Kinan/script.js

```diff
@@ -0,0 +1,450 @@
+/* # Programming Basics: For Loops
+
+These exercises are designed for practising "for" loops. Print all your results to the console.
+
+1. **Addition.** 
+Write a program to add up the numbers 1 to 20. */
+
+for(let i=1;i<=20;i++){
+    console.log(`Number ${i}`)
+}
+//expeacted return
+/* Number 1
+Number 2
+Number 3
+Number 4
+Number 5
+Number 6
+Number 7
+Number 8
+Number 9
+Number 10
+Number 11
+Number 12
+Number 13
+Number 14
+Number 15
+Number 16
+Number 17
+Number 18
+Number 19
+Number 20 */
+/* 
+
+//////////////////////////////////////////////////////////////////////////////////
+2. **There are i bottles of beer on the wall.** 
+Write a program that will output, "There is one bottle of beer on the wall.
+" "There are two bottles of beer on the wall" up until there are five bottles.  */
+//////////////////////////////////////////////////////////////////////////////////
+
+for(i=1;i<=5;i++){
```

Comment:
we need if statement to manage
if one "There is one bottle of beer on the wall."
more than one  "There are two bottles of beer on the wall"